### ISSUE TITLE
Error when pulling crate:2.2.5

### ISSUE THREAD

**Issue by christianbader** [docker-library/official-images#3876]
Hi,

after the last update of CrateDB [6ffaca9](https://github.com/docker-library/official-images/pull/3837/commits/6ffaca97a8719364eba6554b890edd59e274f29f) i get a manifest error when trying to pull `crate:2.2.5`: `Error response from daemon: manifest for crate:2.2.5 not found`
Also `crate:latest` points to the previous version (2.2.4) instead of 2.2.5.

Any ideas?

**Comment by tianon** [comment]
Looking at https://doi-janky.infosiftr.net/job/multiarch/job/amd64/job/crate/, it would appear the latest release has been failing to build due to a GnuPG race condition.  I've kicked off another build manually now to see if we can get lucky and get past it.  We've found that updating `rm -r "$GNUPGHOME"` to `rm -rf "$GNUPGHOME"` doesn't _completely_ remove this particular race, but it does make it less likely to occur (so that's probably a good change to consider making for future updates).
